components/vc/front/include/CANIO_componentSpecific.h

```diff
@@ -86,10 +86,10 @@
 #define set_shockpotVoltFL(m,b,n,s) set(m,b,n,s, shockpot_getFLVoltage())
 #define set_shockpotVoltFR(m,b,n,s) set(m,b,n,s, shockpot_getFRVoltage())
 
-#define set_wheelSpeedLinearFL(m,b,n,s) set(m,b,n,s, wheelSpeed_getSpeedLinear(WHEEL_FL))
-#define set_wheelSpeedLinearFR(m,b,n,s) set(m,b,n,s, wheelSpeed_getSpeedLinear(WHEEL_FR))
-#define set_wheelSpeedRotationalFL(m,b,n,s) set(m,b,n,s, wheelSpeed_getSpeedRotational(WHEEL_FL))
-#define set_wheelSpeedRotationalFR(m,b,n,s) set(m,b,n,s, wheelSpeed_getSpeedRotational(WHEEL_FR))
+#define set_wheelSpeedLinearL(m,b,n,s) set(m,b,n,s, wheelSpeed_getSpeedLinear(WHEEL_FL))
```

Comment:
uhhhh.... what? isn't this gonna be more confusing on CAN now? VCREAR and VCFRONT will both publish `VCxxx_wheelSpeedLinearL`? So you have to use the name of the ECU to figure out if it's the front or rear wheels? I don't like that